#!/usr/bin/env python
PACKAGE = "plane_fitter"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("paint_with_error", bool_t,0,"Paint each point cloud according to each point error", True)
gen.add("max_distance", double_t, 0, "Maxium distance (mm).", 5, 0,500 )
gen.add("min_percentage_of_points", double_t, 0, "Min percentage of inital cloud size.", 5, 0, 100)

exit(gen.generate(PACKAGE, "algorithmParameters", "algorithmParameters"))
